src/pchevrel/Verif.php

```diff
@@ -11,6 +11,7 @@ class Verif
     public $errors = [];
     public $uri;
     public $report_title;
+    public $report;
```

Comment:
I'd prefer not having a property called $report and a public method also called report, that could lead to mistake between calls to $this->report and $this->report()